bundles/CoreBundle/Migrations/Version20200317163018.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20200317163018 extends AbstractPimcoreMigration
+{
+    public function doesSqlMigrations(): bool
+    {
+        return false;
+    }
+
+    /**
+     * @param Schema $schema
+     *
+     * @throws \Exception
+     */
+    public function up(Schema $schema)
+    {
+        $list = new DataObject\ClassDefinition\Listing();
+        $list = $list->load();
+
+        foreach ($list as $class) {
+            $this->writeMessage(sprintf('Saving class: %s', $class->getName()));
+            $class->save();
```

Comment:
I am wondering, since this actually changes the class-definition file as well, wouldn't it be more sufficient to just re-generate the source-code files and done?

that doesn't just affect this PR, but also all future ones.

Reply:
@dvesh3 there is an option for this

    public function ClassDefinition::save($saveDefinitionFile = true)

Reply:
Cool, yes! Let's do that!  @dvesh3 can you please update the PR once again? Thanks a lot 🙏

Reply:
Just for clarification, maybe we should introduce a new function: `generatePhpClassFiles` or something. Maybe also introduce a service responsible for class generation. Guess that has already been propsoed?

Reply:
Not a bad idea, but out of scope for this PR. 😊
Hmm, can't remember a proposal for this, but anyway, like I said, not a bad idea at all 😉

Reply:
Just saw that the parameter doesn't help in this case, so we need to move the code into another public method ...

Reply:
Introduced new public method `generateClassFiles` and now migration calls this new method instead of save.

Reply:
👍 Would be fine to have that in a separate PR, but fine for me :)

Thanks @dvesh3

Reply:
Thanks for review guys 🙌